mk_server/monkey.c

```diff
@@ -122,6 +122,27 @@ struct mk_server *mk_server_create()
         return NULL;
     }
 
+    /* Library mode: start event loop */
+    server->lib_evl_start = mk_event_loop_create(1);
+    if (!server->lib_evl_start) {
+        mk_mem_free(server);
```

Comment:
This call is missing here `mk_event_loop_destroy(server->lib_evl);`

Reply:
Nice catch thanks! Fixed in [efd10da](https://github.com/monkey/monkey/pull/381/commits/efd10dabdd2012dc6bff74691bb833ef62a16e2e)